mw-webapp/src/logic/waysTable/OwnWaysTable.tsx

```diff
@@ -42,9 +52,19 @@ export const OwnWaysTable = (props: PropsWithUuid) => {
   });
 
   return (
-    <WaysTable
-      data={ownWays}
-      columns={ownWaysTableColumns}
-    />
+    <>
+      {props.uuid ?
```

Comment:
here too would be better to replace with &&

Reply:
Changed